### DISCUSSION TITLE
Neper/FEPX to study liquid metal embrittlement

### DISCUSSION THREAD

**Discussion by okigami** [neperfepx/neper#213] [Show and tell]
Hi all!

I'm looking for a tutorial for beginners.
In my PhD I'm studying liquid metal embrittlement of a steel by Zn.
Most of the literature does Gleeble hot tensile test to 'assess a critical stress'.
But honestly, this will only give a very rough estimation of a tensile stress needed to initiate a crack by liquid Zinc.
I'm looking for a solution using Neper/FEPX to model a hot tensile test and study grain decohesion by looking at local stresses.
The idea is to connect a FEA model (of an RSW process) with Thermo-Calc to get to the phase formation and reaction kinetics.
Then will feed that to the crystal plasticity model to get a more accurate prediction of what happens at the micro level.

Any critics and/or suggestions are very welcome!

**Comment by rquey** [comment]
Thanks for sharing. I don't know about the simulation of the RSW process, but you should be able to use Neper/FEPX for the polycrystal plasticity part.

Maybe others have inputs.
